src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -135,6 +135,23 @@ public function create_item_permissions_check( $request ) {
 			);
 		}
 
+		// If the upload is an image, check if the server can handle the mime type.
```

Comment:
Could we add a way to disable this behavior, e.g. with a filter?

With client-side image processing I need to be able to upload AVIF images and the like without any additional server-side processing.

Reply:
That is a good suggestion, I was wondering about the same thing. If we do add a filter, I feel like it should impact both editor and media uploads, what do you think?

Reply:
That makes sense to me, yes.